### DISCUSSION TITLE
[Config Support]: Home Assistant live view available, even without go2rtc

### DISCUSSION THREAD

**Discussion by ronbruins** [blakeblackshear/frigate#10552] [Config Support] [answered]
### Describe the problem you are having

I configured go2rtc,as it is stated that this is needed for live view in Home Assistant. But when I go to frigate integration in HA, I can click on the camera and it shows me a live stream, even without go2rtc. Why should I Use go2rtc anyway?

### Version

latest

### Frigate config file

```yaml
NA
```

### Relevant log output

```shell
NA
```

### Frigate stats

```json
NA
```

### Operating system

Other Linux

### Install method

Docker Compose

### Coral version

USB

**Comment by NickM-27** [comment, marked as answer]
It's not actually a live view it's just a live snapshot that updates once per second and of course doesn't support audio

**Reply by ronbruins** [reply]
OK, that's not easy to spot unless there's actual movement. Also in HA it always says recording as well, is that normal? 
So for a 'real' live stream it needs go2rtc

**Reply by NickM-27** [reply]
Yes, if recording is enabled then it will say recording.

**Reply by ronbruins** [reply]
Thanks, makes sense :) so bottom line is if I configure go2rtc my live view will be an actual live view instead the stop motion way it is right now? And will HA pick this up automatically once I have changed the configuration to a go2rtc one?

**Reply by NickM-27** [reply]
yes, that is what go2rtc provides. Assuming it is setup correctly, the frigate integration will provide the stream in HA

**Reply by ronbruins** [reply]
Well, I setup go2rtc and works fine in frigate and see the MSE stream. In HA I still get the stutter 1 per second (and see the logs filled with providing the latest.jpg) no matter what I do. how to troubleshoot this?

**Reply by NickM-27** [reply]
I'd suggest making a new issue and filling out the template

**Comment by ronbruins** [comment]
Thanks for the info, closing this one as it is answered
